src/renderers/webgl/WebGLUniforms.js

```diff
@@ -414,7 +414,7 @@ function setValueT6( gl, v, textures ) {
 
 }
 
-// Integer / Boolean vectors or arrays thereof (always flat arrays)
+// Integer / Boolean
```

Comment:
Um, why have to deleted `vectors or arrays thereof (always flat arrays)`?

Reply:
This comment seems to be directly copied from #L607, where the code about array occurs, and the code here is only about single integer / bool (vectors at #L431), so I changed the incorrect comment.

Reply:
The comment refers to all call `setValueV*i()` calls. Starting from `setValueV2i()`, `v` is an array and `arraysEqual()` is used.

Reply:
I suppose the "arrays" in the comment refers to `setValueV*iArray`, aka  `uniform uvec2 foo[42];`. The "always flat arrays" is a contrast to "flat or from THREE classes": `flatten()` in `setValueV*fArray`. I think it is a mistake.

Reply:
Using git blame I found that the mistake is from [this](https://github.com/mrdoob/three.js/commit/22c2356853bcd06a0912dd470056a34a4012b86e#diff-ae819466109851758b72993ae3f4d77a3a1835d343ddf1f9dfa0c7a791b65a6cR504) commit.
The `setValue*iv` is splited into `setValueV*i` and `setValueV*ia`, then the comment is copied twice but forgett to change.

Reply:
Please use the same pattern like `setValueV*f()` is documented.

Reply:
In the [next](https://github.com/mrdoob/three.js/pull/21316/commits/98b3de4a23a71c128c91ddf0084417535e772c69) commit, the comments are rewritten, and the order of the `setValue*` funcitons is also changed to fit their order in `getSingularSetter` and `getPureArraySetter`.